_index.md

```diff
@@ -0,0 +1,50 @@
+Here is the template we will use for the _index.md file
```

Comment:
I think this line should be deleted "Here is the template...". The file should start with `---` as the first line.